ui/packages/shared/parser/src/index.ts

```diff
@@ -249,12 +308,12 @@ export class Query {
       // We're not going to skip suggesting to type a whitespace character.
       if (!(nextSymbol.type !== undefined && nextSymbol.type === 'space')) {
         if (nextSymbol.literal !== undefined) {
-          const suggestion = {type: 'literal', value: nextSymbol.literal as string};
+          const suggestedValue = nextSymbol.literal as string;
           if (
-            suggestions.findIndex(s => s.type === 'literal' && s.value === suggestion.value) === -1
+            suggestions.findIndex(s => s.type === 'literal' && s.value === suggestedValue) === -1
```

Comment:
If we save the suggestion types as an enum (as suggested here: https://github.com/parca-dev/parca/pull/2045/files#r1015431013) we could reuse those values in the following manner:

```suggestion
            suggestions.findIndex(s => s.type === SuggestionType.Literal && s.value === suggestedValue) === -1
```

Reply:
Yeah this would only work if we used the enum, but based on the above comments, this is not actually duplicate, as there would be a type error if we misspelled this.